pkg/api/utils/apiUtils.go

```diff
@@ -124,7 +76,7 @@ func (a *APIHandler) CreateProject(project models.CreateProject) (string, *model
 	if err != nil {
 		return "", buildErrorResponse(err.Error())
 	}
-	return post(a.Scheme+"://"+a.getBaseURL()+"/shipyard-controller/v1/project", bodyStr, a)
+	return post(a.Scheme+"://"+a.getBaseURL()+"/"+shipyardControllerBaseURL+"/v1/project", bodyStr, a)
```

Comment:
can you also make "/v1/project" a costant named e.g. `v1ProjectPath`.
same for "/v1/events/, "/v1/metadata" and so on